app/models/shipit/stack.rb

```diff
@@ -386,16 +391,21 @@ def active_task
     end
 
     def locked?
+      lock_reason.present? && lock_level != LOCK_LEVEL_ADVISORY
+    end
+
+    def soft_locked?
```

Comment:
It's not clear to me without looking at the implementation whether `soft_locked?` means "locked, and level == advisory", or just "locked, and level is at least advisory".

Reply:
I'm going to rename the method to `advisory_locked?`.